### PR TITLE
v1.1.3: Upgrade bosh to 255.3 and cf-release to 231.

### PR DESCRIPTION
**Pull request by AbelHu**
### Changelog
* Upgrade bosh to 255.3
* Upgrade cf-release to 231

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@AbelHu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (abelch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: bosh-setup/manifests/multiple-vm-cf.yml [hunk 1]
```diff
@@ -320,6 +456,153 @@ properties:
...
     - {name: ccadmin, password: c1oudc0w, tag: admin}
+  consul:
+    agent:
+      log_level: null
+      servers:
+        lan:
+        - REPLACE_WITH_CONSUL_IP
+    agent_cert: |+
+      -----BEGIN CERTIFICATE-----
+      MIIEITCCAgugAwIBAgIQTDMCJHBiQrY8hlM78doHTzALBgkqhkiG9w0BAQswEzER
```
Comment (singhkay) [inline bosh-setup/manifests/multiple-vm-cf.yml]: Is it ok to have private keys/certs like this?

Comment (AbelHu) [inline bosh-setup/manifests/multiple-vm-cf.yml]: They are copied from community manifests and only can be accessed by internal trusted components. And these examples are for testing but not for production.

Best Regards,
Abel HU

? 2016?3?10??16:56?Kay Singh <notifications@github.com<mailto:notifications@github.com>> ???


In bosh-setup/manifests/multiple-vm-cf.yml<https://github.com/Azure/azure-quickstart-templates/pull/1371#discussion_r55649462>:

> @@ -320,6 +456,153 @@ properties:
>      port: 5524
>      roles:
>      - {name: ccadmin, password: c1oudc0w, tag: admin}
> +  consul:
> +    agent:
> +      log_level: null
> +      servers:
> +        lan:
> +        - REPLACE_WITH_CONSUL_IP
> +    agent_cert: |+
> +      -----BEGIN CERTIFICATE-----
> +      MIIEITCCAgugAwIBAgIQTDMCJHBiQrY8hlM78doHTzALBgkqhkiG9w0BAQswEzER


Is it ok to have private keys/certs like this?

-

Comment (singhkay) [inline bosh-setup/manifests/multiple-vm-cf.yml]: Got it

> And these examples are for testing but not for production.

Can you leave something regarding this in the readme file?

Comment (AbelHu) [inline bosh-setup/manifests/multiple-vm-cf.yml]: Actually in created VMs, we put these files under ~/example_manifests. For cloud foundry users, these files are for reference. If you think it is still not enough, we can add comments here.

Comment (singhkay) [inline bosh-setup/manifests/multiple-vm-cf.yml]: Yeah let's leave comments or hints as to the nature of these and where they are used to remove any confusion for users/security team not familiar with Cloudfoundry/Bosh.

Comment (AbelHu) [inline bosh-setup/manifests/multiple-vm-cf.yml]: Done.

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**